CIP-0013/CIP-0013.md

```diff
@@ -43,27 +54,64 @@ Example:
 
 ## ABNF Grammar (Proposal)
 
-This is an initial, simplified protocol definition for fast implementation; it only requires an address and an optional amount parameter. As discussed above, these rules are likely to evolve in time in order to support additional features, including unique capabilities of the Cardano blockchain.
+This is an initial, simplified protocol definition for fast implementation; it only requires:
+
+* for a payment URI (authority unspecified), an address and an optional amount parameter;
+* for a stake pool URI (authority = `stake`), a single stake pool reference.
+
+As discussed above, these rules are likely to evolve in time in order to support additional features, including multiple stake pool references (in proportions defining a "portfolio") and other unique capabilities of the Cardano blockchain.
 
 ```
-cardanourn     = "web+cardano:" cardanoaddress [ "?" amountparam ]
+cardanourn = "web+cardano:" (paymentref | stakepoolref)
+
+paymentref = cardanoaddress [ "?" amountparam ]
 cardanoaddress = *(base58 | bech32)
-amountparam    = "amount=" *digit [ "." *digit ]
+amountparam = "amount=" *digit [ "." *digit ]
+
+stakepoolref = "//stake?" stakepool
+stakepool = poolhexid | poolticker
+poolhexid = 56HEXDIG
+poolticker = 3*5UNICODE
 ```
 
+### Payment URI queries
+
 The amount parameter must follow the [same rules](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki#transfer-amountsize) described in BIP-21, namely, it must be specified in decimal ADA, without commas and using the period (.) decimal separator.
 
+### Stake pool URI queries
+
+For brevity, essential in many Internet contexts, `poolticker`  must be supported here in addition to the unambiguous `poolhexid`.  A safe interpretation of `poolticker` should be assured by the Daedalus wallet's integration with [SMASH](https://github.com/input-output-hk/smash) metadata.
+
+When there is more than one pool registered with the specified `poolTicker` (whether for pool groups which have the same ticker for all pools, or for separate pools using the same ticker), the choice to which pool to finally delegate is left to the user through the wallet UI.
+
+In all cases above, the wallet UI must select, out of multiple wallets, which wallet(s) the user will be delegating from, and should confirm the exact choice even when it is unambiguous from the URI.
+
+These yet unsupported parameters in the URI query string should (by preference of the wallet UI designers) *either* be ignored *or* generate a warning message, to avoid leading the user to believe they are implementing a currently unsupported but perhaps popularly referenced multi-pool delegation list:
```

Comment:
Perhaps this paragraph should be in a standalone header instead of under the "Stake pool URI queries" section?

Reply:
Yes, you're right, these are more about the wallet behaviour than the URI query itself.  These have been edited for clarity a bit and titled as **Handling stake pool links**.